setup.json

```diff
@@ -19,6 +19,17 @@
         ],
         "aiida.cmdline.data": [
             "restapi = aiida_restapi.cli:data_cli"
+        ],
+        "aiida.rest.post": [
+            "data.str.Str.| = aiida_restapi.models:Node",
+            "data.float.Float.| = aiida_restapi.models:Node",
+            "data.int.Int.| = aiida_restapi.models:Node",
+            "data.bool.Bool.| = aiida_restapi.models:Node",
+            "data.structure.StructureData.| = aiida_restapi.models:Node",
+            "data.orbital.OrbitalData.| = aiida_restapi.models:Node",
+            "data.list.List.|  = aiida_restapi.models:Node",
+            "data.dict.Dict.| = aiida_restapi.models:Node",
+            "data.singlefile.SingleFileData.| = aiida_restapi.models:Node"
```

Comment:
Specifying entry points in the setup.json/setup.py is the right approach for AiiDA plugins but here it is not ideal since we are adding entry points here that depend on a different package (aiida-core).
I.e. this list would need to be different for any AiiDA version that has different versions of these data plugins.

It seems to be possible to add entry points dynamically, see 
https://stackoverflow.com/a/48666503/1069467
This uses `pkg_resources`- perhaps there is a way to do it with importlib as well (not necessarily; if I remember well, in your first PR to AiiDA you already encountered an issue where importlib did lack some of the functionality of the old `pkg_resources`)

In any case, this does seem like a use case for dynamically adding entry points (i.e. it could be done in the `__init__.py` of aiida_restapi)

Reply:
I guess this list can be removed now?

Didn't we discuss that we would merge things between the full_type strings known to AiiDa and entrypoints that could be added here?